ext/reflection/php_reflection.c

```diff
@@ -916,6 +919,8 @@ static zval *property_get_default(zend_property_info *prop_info) {
 		zval *prop = &ce->default_static_members_table[prop_info->offset];
 		ZVAL_DEINDIRECT(prop);
 		return prop;
+	} else if (prop_info->flags & ZEND_ACC_VIRTUAL) {
+		return NULL;
```

Comment:
Nit: For all purposes of this function, there's no distinction between undef and virtual. So, I propose
```suggestion
		return &EG(uninitialized_zval);
```
And simplifying the NULL checks away on its usages.

Reply:
`EG(uninitialized_zval)` is actually initialized to `IS_NULL` rather than `IS_UNDEF`. We could add a new global, but let's do that in a separate PR if desired.

Reply:
Oh, I meant `EG(error_zval)`. Yes, uninitialized is IS_NULL.

Reply:
`EG(error_zval)` is `_IS_ERROR`, also not `IS_UNDEF`.

Reply:
Yes, but you can check for it, anyway, it doesn't matter enough.